src/js/pages/jobs/JobDetailPage.js

```diff
@@ -143,21 +149,34 @@ class JobDetailPage extends mixin(StoreMixin, TabsMixin) {
   }
 
   getActionButtons() {
-    let dropdownItems = [
-      {
-        className: 'hidden',
-        html: 'More',
-        id: JobActionItem.MORE
-      },
-      {
+    let job = MetronomeStore.getJob(this.props.params.id);
+    let dropdownItems = [];
+    let [schedule] = job.getSchedules();
+
+    dropdownItems.push({
+      className: 'hidden',
+      html: 'More',
+      id: JobActionItem.MORE
+    });
+
+    if (schedule != null && schedule.enabled) {
```

Comment:
I find it easier to follow!
And I doubt that this is increasing cyclomatic complexity, I guess it remains the same.
On a binary level yours have 2 x 1 levels = 2.
Mine has 1 x 2 levels = 2.